cli/src/command/maintenance.rs

```diff
@@ -62,13 +69,32 @@ impl Action {
         db.merge_sync(fork.into_patch())?;
         Ok(())
     }
+
+    fn restart_migration(
+        node_config: &Path,
+        db_path: &Path,
+        service_name: &str,
+    ) -> Result<(), Error> {
+        let node_config: NodeConfig = load_config_file(node_config)?;
+        let db: Box<dyn Database> = Box::new(RocksDB::open(
+            db_path,
+            &node_config.private_config.database,
+        )?);
+        let fork = db.fork();
+
+        remove_local_migration_result(&fork, service_name);
```

Comment:
I've understood just now (this is what you get when you rush to a release, I suppose 🤦‍♂): we need to remove migration data via `rollback_migration`, don't we? Otherwise, the restarted migration script (if it's written correctly) will do nothing and finish immediately with supposedly the same result. Ideally, this should be tested (for example, via a check in the script that no data is set in the migration).

Reply:
So, in `remove_local_migration_result` we can call `Schema::add_migration_rollback` and this will remove data and local result, am I right?